content/webapp/app.js

```diff
@@ -43,6 +44,33 @@ module.exports = app
     server.use(middleware);
     server.use(bodyParser());
 
+    routeIfToggleOn(
+      '/covid-we-are-open',
+      '/covid-we-are-open',
+      router,
+      app,
+      {},
+      'buildingReopening'
+    );
+
+    routeIfToggleOn(
+      '/covid-keeping-you-safe',
+      '/covid-keeping-you-safe',
+      router,
+      app,
+      {},
+      'buildingReopening'
+    );
+
+    routeIfToggleOn(
+      '/covid-book-your-ticket',
+      '/covid-book-your-ticket',
+      router,
+      app,
+      {},
+      'buildingReopening'
+    );
```

Comment:
Think I got it.

Could we have something like `renderIfToggleOn`? As the route exists, it just returns 404.